RELEASE.md

```diff
@@ -239,6 +240,16 @@ After the release is complete do the following:
       - `go-uretprobe-counter-install-selinux.yaml`
       - `go-kprobe-counter-install-selinux.yaml`
       - `go-app-counter-install-selinux.yaml`
+- Sign the release tar files
+  - Download the `bpfman-linux-*.tar.gz` files.
+  - Sign the files with the following command (GPG key required):
```

Comment:
What I mean is that we probably shouldn't start signing releases until such times as we understand how we're managing the release signing keys. GPG is probably not the best option (IMHO) and scorecard supports multiple formats. See: https://github.com/ossf/scorecard/blob/49c0eed3a423f00c872b5c3c9f1bbca9e8aae799/docs/checks.md#signed-releases

Reply:
I used my own key to sign them using the instructions from the link you gave that said:
> If the source is hosted on GitHub, check out the steps [here](https://wiki.debian.org/Creating%20signed%20GitHub%20releases).

This satisfied the scorecard scan to the tune of 8/10 points, but I was wondering if we should/could use one of the more preferred options and whether there was a way to automate the process.

Reply:
Yes. In-Toto is the recommended way. GitHub has started adding support for this natively and it uses Sigstore for signatures https://docs.github.com/en/actions/security-for-github-actions/using-artifact-attestations/using-artifact-attestations-to-establish-provenance-for-builds

It might require some experimentation to get right though, but can be easily automated - and more importantly doesn't require us messing with GPG.

Reply:
I removed the signing instructions until we work out a better process.